po/de.po

```diff
@@ -4287,16 +4246,16 @@ msgid ""
 "locations when shadowing an immutable variable in a scope, even if the type "
 "does not change."
 msgstr ""
-"* Definition: Shadowing unterscheidet sich von Mutation, da nach dem "
-"Shadowing die Speicherplätze beider Variablen gleichzeitig existieren. Beide "
-"sind unter demselben Namen verfügbar, je nachdem, wo Sie ihn im Code "
-"verwenden.\n"
+"* Definition: Verschattung (shadowing) unterscheidet sich von Mutation, da "
+"nach dem Shadowing die Speicherplätze beider Variablen gleichzeitig "
```

Comment:
```suggestion
"nach der Verschattung die Speicherplätze beider Variablen gleichzeitig "
```

Reply:
How often should we write the english translation in parentethes after the keyword?

Reply:
Hey @Throvn, I don't think we have a standard for this across translations — we've let the different language groups decide this for themselves. If we should give general advice, then we should put it in [STYLE.md](https://github.com/google/comprehensive-rust/blob/main/STYLE.md#translations).

@henrif75, you might be interested in this too.